.gitignore

```
@@ -0,0 +1,6 @@
+node_modules
+yarn-error.log
+tsconfig.json
+src/**
+.eslintrc.js
+.eslintignore
```

End file with new line (there's aN ESLINT rule for that as well ;) )
```suggestion
.eslintignore

```